app/main/db/queries.py

```diff
@@ -34,10 +39,30 @@ def browse_data(
     elif consignment_id:
         consignment = Consignment.query.get_or_404(consignment_id)
         validate_body_user_groups_or_404(consignment.consignment_bodies.Name)
-        browse_query = _build_consignment_filter_query(consignment_id)
+
```

Comment:
Could we set the default to `browse_data` to `date_filter_field="date_last_modified"`

then this whole block from line 43 to 52 (inclusive) could be:

```
consignment_view_filters = {"date_range": date_range, "date_range_field": date_filter_field.lower()} if date_range else {}
browse_query = _build_consignment_filter_query(consignment_id, **consignment_view_filters)
```